html_xblock/static/js/html.js

```diff
@@ -0,0 +1,78 @@
+/* Javascript for HTMLXBlock. */
+
+const CUSTOM_FONTS = "Default='Open Sans', Verdana, Arial, Helvetica, sans-serif;";
+const STANDARD_FONTS = "Andale Mono=andale mono,times;" + "Arial=arial,helvetica,sans-serif;" + "Arial Black=arial black,avant garde;" + "Book Antiqua=book antiqua,palatino;" + "Comic Sans MS=comic sans ms,sans-serif;" + "Courier New=courier new,courier;" + "Georgia=georgia,palatino;" + "Helvetica=helvetica;" + "Impact=impact,chicago;" + "Symbol=symbol;" + "Tahoma=tahoma,arial,helvetica,sans-serif;" + "Terminal=terminal,monaco;" + "Times New Roman=times new roman,times;" + "Trebuchet MS=trebuchet ms,geneva;" + "Verdana=verdana,geneva;" + "Webdings=webdings;" + "Wingdings=wingdings,zapf dingbats";
+const FONTS = CUSTOM_FONTS + STANDARD_FONTS;
+
+const PLUGINS = ["codesample", "image", "link", "lists", "textcolor", "codemirror"];
+const PLUGINS_DIR = "/resource/html5/public/plugins/";
```

Comment:
This URL doesn't seem to work; in Studio in the editor, I'm seeing these sort of errors in the JS console:

```
http://localhost:18010/resource/html5/public/plugins/codemirror/plugin.min.js 404 (Not Found)
```

Since there is no XBlock JavaScript API to get a resource URL, you should call `runtime.local_resource_url()` as described in `html_xblock/static/README.txt` and then pass the resulting URLs to JavaScript via the second argument to `frag.initialize_js(function, data)` which will then get passed in to `function HTML5XBlock(runtime, element, data)` as the third argument.